bpf/node_config.h

```diff
@@ -39,6 +39,7 @@ DEFINE_IPV6(HOST_IP, 0xbe, 0xef, 0x0, 0x0, 0x0, 0x0, 0x0, 0x1, 0x0, 0x0, 0xa, 0x
 #define CT_CONNECTION_LIFETIME_NONTCP	60
 #define CT_SERVICE_LIFETIME_TCP		21600
 #define CT_SERVICE_LIFETIME_NONTCP	60
+#define CT_SERVICE_CLOSE_WAIT_BEFORE_REBALANCE 30
```

Comment:
nit: `CT_SERVICE_CLOSE_WAIT_BEFORE_REBALANCE` maybe lets shorten it a bit `CT_SERVICE_CLOSE_REBALANCE`